js/components/selector.js

```diff
@@ -84,7 +84,11 @@ export default {
 
     onShow: function () {
       setTimeout(() => { // timeout is a hack to make focus work in Chrome
-        this.$refs.choices.find(choice => choice.selected).$refs.input[0].focus()
+        if (this.$refs.choices.find(choice => choice.selected)) {
+          this.$refs.choices.find(choice => choice.selected).$refs.input[0].focus()
```

Comment:
Can we pull out the `find` call to reduce repetition:

```suggestion
       const selected = this.$refs.choices.find(choice => choice.selected))
        if (selected) {
          selected.$refs.input[0].focus()
```